README.md

```diff
@@ -6,7 +6,17 @@ Basic idea of this application is to create a platform between kirana/retail sto
 This application has two faces, first is for kirana/retail store owner and second is for customers.
 
 # Kirana/Retail store owner goals:
-Here, owner can create an account and list down the products present in the store.
+
+We have two major goals to reach in this area. 
```

Comment:
Can we have file other than Readme, to track this design info. 
Readme file gonna be the public facing info about the application but not the info

Reply:
yes we can do it. I will move it.